cli/data.py

```diff
@@ -179,18 +193,43 @@ def update(
     #     pathlib.Path("data/pre-agg-wide-dates.csv"), pathlib.Path("data/pre-agg-static.csv")
     # )
     if aggregate_to_country:
-        country_dataset = region_aggregation.aggregate_regions(
-            multiregion_dataset,
-            pipeline.us_states_and_territories_to_country_map(),
-            reporting_ratio_required_to_aggregate=DEFAULT_REPORTING_RATIO,
-        )
-        multiregion_dataset = multiregion_dataset.append_regions(country_dataset)
+        multiregion_dataset = run_aggregate_to_country(multiregion_dataset)
         multiregion_dataset.print_stats("aggregate_to_country")
 
     combined_dataset_utils.persist_dataset(multiregion_dataset, path_prefix)
     multiregion_dataset.print_stats("persist")
 
 
+def _log_unexpected_aggregated_variables_to_drop(variables_to_drop: pd.Index):
+    unexpected_drops = variables_to_drop.difference(US_AGGREGATED_EXPECTED_VARIABLES_TO_DROP)
+    if not unexpected_drops.empty:
+        log = structlog.get_logger()
+        log.warn(
+            US_AGGREGATED_VARIABLE_DROP_MESSAGE, variable=unexpected_drops.to_list(),
+        )
+
+
+def run_aggregate_to_country(dataset_in: timeseries.MultiRegionDataset):
```

Comment:
Feels like the `run_aggregate_to_country` should be in `libs/datasets/usa_aggregation.py` or isn't there a `custom_aggregations` module? This file doesn't have too much of the guts in it and probably best to keep it that way